pkg/kubefedctl/util/util.go

```diff
@@ -128,10 +128,16 @@ func IsFederatedAPIResource(kind, group string) bool {
 
 // GetNamespace returns namespace of the current context
 func GetNamespace(hostClusterContext string, kubeconfig string, config FedConfig) (string, error) {
-	ns, _, err := config.GetClientConfig(hostClusterContext, kubeconfig).Namespace()
+	clientConfig := config.GetClientConfig(hostClusterContext, kubeconfig)
+	currentContext, err := options.CurrentContext(clientConfig)
+	if err != nil {
+		return "", err
+	}
+
+	ns, _, err := clientConfig.Namespace()
 	if err != nil {
 		return "", errors.Wrapf(err, "Failed to get ClientConfig for host cluster context %q and kubeconfig %q",
-			hostClusterContext, kubeconfig)
+			currentContext, kubeconfig)
```

Comment:
This seems only for message correctness. If that's true, do we plan to fix in `orphaning.go` https://github.com/kubernetes-sigs/kubefed/blob/0eeda018624acdbf6fc28ef0b67d2e3e8b6e614b/pkg/kubefedctl/orphaning/orphaning.go#L105

Reply:
Done